File: Cano-Rodriguez-Claudio/Procesos/Python/thread.py
import threading
import os
import time

def child_thread():
    while True:
        print(f"\t\t\tH={os.getpid()}")
        time.sleep(0.5)  # Optional delay

if __name__ == "__main__":
    # Create and start the child thread
    thread = threading.Thread(target=child_thread)
    thread.start()

    # Main thread
    while True:
        print(f"\tP={os.getpid()}")
        time.sleep(0.5)  # Optional delay

File: Cano-Rodriguez-Claudio/Procesos/Go/thread.go
package main

import (
	"fmt"
	"os"
	"time"
)

func childThread() {
	for {
		fmt.Printf("\t\t\tH=%d\n", os.Getpid())
		time.Sleep(500 * time.Millisecond) // Optional delay
	}
}

func main() {
	// Start a goroutine (child thread)
	go childThread()

	// Main thread
	for {
		fmt.Printf("\tP=%d\n", os.Getpid())
		time.Sleep(500 * time.Millisecond) // Optional delay
	}
}
